### PR TITLE
Renaming

### PR DESCRIPTION
**Pull request by pipo02mix**
Towards https://github.com/giantswarm/giantswarm/issues/20218#issuecomment-1020158166

This PR:

- adds/changes/removes etc

### Testing

Description on how rbac can be tested.

- [x] fresh install works
  - [ ] AWS
  - [ ] Azure
  - [ ] KVM
- [x] upgrade from previous version works
  - [ ] AWS
  - [ ] Azure
  - [ ] KVM

#### Other testing

Description of features to additionally test for rbac installations.

- [x] check reconciliation of existing resources after upgrading
- [x] X still works after upgrade
- [ ] Y is installed correctly

### Checklist

- [ ] Update changelog in CHANGELOG.md.
- [ ] Make sure `values.yaml` and `values.schema.json` are valid.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -8,10 +8,10 @@ workflows:
       - architect/push-to-app-catalog:
           context: "architect"
           executor: "app-build-suite"
-          name: "package and push rbac chart"
+          name: "package and push rbac bootstrap chart"
```
Comment (marians) [inline .circleci/config.yml:11]:
```suggestion
          name: "package and push rbac-bootstrap chart"
```

File: README.md [hunk 2]
```diff
@@ -1,14 +1,22 @@
-[![CircleCI](https://circleci.com/gh/giantswarm/rbac-app.svg?style=shield)](https://circleci.com/gh/giantswarm/rbac-app)
+[![CircleCI](https://circleci.com/gh/giantswarm/rbac-bootstrap-app.svg?style=shield)](https://circleci.com/gh/giantswarm/rbac-bootstrap-app)
 
-# rbac chart
+# RBAC bootstrap chart
 
-Giant Swarm offers a rbac App which can be installed in workload clusters.
+Giant Swarm offers a RBAC bootstrap App as a configuration package to bootstrap the initial role bindings.
```
Comment (marians) [inline README.md:5]:
```suggestion
Giant Swarm offers the RBAC bootstrap app as a configuration package to bootstrap the initial role bindings for a workload cluster.
```

File: README.md [hunk 3]
```diff
@@ -1,14 +1,22 @@
...
+
+- You can configure the list of users and groups that will have access to the cluster initially.
+
 **Why did we add it?**
+
+- Some customers want to define a set of bindings as a base to every cluster in order to enable their automation and admin to get quick access to the workload cluster API.
+
 **Who can use it?**
 
+- This app run in all clusters and versions.
```
Comment (marians) [inline README.md:18]:
```suggestion
- This app can be used in all clusters and release versions.
```

File: README.md [hunk 4]
```diff
@@ -23,9 +31,25 @@ There are 3 ways to install this app onto a workload cluster.
 **This is an example of a values file you could upload using our web interface.**
 ```
```
Comment (marians) [inline README.md:32]:
```suggestion
```yaml
```

File: README.md [hunk 5]
```diff
@@ -23,9 +31,25 @@ There are 3 ways to install this app onto a workload cluster.
...
+      - ns1
+      - ns2
+  - role: admin
+    users:
+      - admin@myco.com
+    groups:
+      - adminteam
 ```
 
+This configuration create role bindings for `editor1@myco.com` and `editor1@myco.com` in the `ns1` and `ns2` namespaces. At the same time it creates a cluster role binding for `admin@myco.com` and `adminteam` to `admin` cluster role.
```
Comment (marians) [inline README.md:51]:
```suggestion
This configuration creates role bindings for `editor1@myco.com` and `editor1@myco.com` in the `ns1` and `ns2` namespaces. At the same time it creates a cluster role binding for `admin@myco.com` and `adminteam` to the `admin` cluster role.
```

File: helm/rbac-bootstrap/Chart.yaml [hunk 6]
```diff
@@ -0,0 +1,10 @@
+apiVersion: v2
+appVersion: 0.0.1
+annotations:
+  application.giantswarm.io/team: "rainbow"
+  config.giantswarm.io/version: 1.x.x
+name: rbac-bootstrap
+description: A Helm chart to facilitate the initial configuration of role bindings
```
Comment (marians) [inline helm/rbac-bootstrap/Chart.yaml:7]:
```suggestion
description: Helps to set up initial role bindings in a new cluster
```

**Comment by marians** [comment]
Thank you @pipo02mix !

**Review by anvddriesch — APPROVED** [review]

**Review by marians — APPROVED** [review]

**Pull request merged**